datalake/src/main/java/com/sequenceiq/datalake/flow/loadbalancer/dns/UpdateLoadBalancerDNSActions.java

```diff
@@ -64,6 +75,7 @@ protected void doExecute(SdxContext context, StartUpdateLoadBalancerDNSEvent pay
                 updateLoadBalancerDNSService.performLoadBalancerDNSUpdate(sdxCluster);
```

Comment:
I'm just leaving a note that this should be invoked from a handler as it seems to be doing a lot of stuff in the background. I'm not even sure that this should happen in request time

Reply:
Should we create a JIRA for this?

Reply:
+1, this likely belongs to a handler as it makes a web service call to `StackV4Endpoint`. Could you please raise a ticket tracking this?

Reply:
Created: https://jira.cloudera.com/browse/CB-17913